AzureWebFarm.OctopusDeploy/Infrastructure/OctopusDeploy.cs

```diff
@@ -106,6 +131,11 @@ public static IOctopusRepository GetRepository(ConfigSettings config)
 
         public void UninstallTentacle()
         {
+            var machineResource = _repository.Machines.FindByName(_machineName);

```

Comment:
I found that just uninstalling the tentacle didn't remove the machine from the ocotpus server. It just gave me an unresponsive machine that was holding up deployments. This makes sure it is deleted from the server.